modules/back-end/src/Domain/Utils/StringHelper.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Text.RegularExpressions;
+
+namespace Domain.Utils;
+
+public static class StringHelper
+{
+    /// <summary>
+    /// This regular expression matches any single character that is one of the following special characters:
+    /// . * + ? ^ = ! : $ { } ( ) | [ ] / \
+    /// It can be used to search for any of these special characters in a given text or string.
+    /// </summary>
```

Comment:
I think this summary is not clear enough, how about

```csharp
/// <summary>
/// Escapes a set of characters (. * + ? ^ = ! : $ { } ( ) | [ ] / \) by replacing them with their escape codes.
/// This converts a string so that it can be used as a constant within a regular expression safely.
/// </summary>
```

Reply:
Sure. Thats the best ChatGPT could give me lol.